config/toml.go

```diff
@@ -49,6 +49,14 @@ light-client-primary-addr = "{{ .Panacea.LightClientPrimaryAddr }}"
 # Witness addresses (comma-separated) for light client verification
 
 light-client-witness-addrs= "{{ StringsJoin .Panacea.LightClientWitnessAddrs "," }}"
+
+###############################################################################
+###                         Ipfs Configuration                           ###
+###############################################################################
+
+[ipfs]
+
+ipfs-addr = "{{ .Ipfs.IpfsNodeAddr }}"
```

Comment:
```suggestion
ipfs-node-addr = "{{ .Ipfs.IpfsNodeAddr }}"
```

Reply:
Fixed in [e22cb5a](https://github.com/medibloc/panacea-doracle/pull/88/commits/e22cb5ac922d2ee9ceb2e1be692480ae83f0edda).